mysql-test/suite/rpl/t/rpl_ddl_high_priority.test

```diff
@@ -0,0 +1 @@
+--source include/rpl_ddl_high_priority.inc
```

Comment:
Why this additional level of indirection. Can the content of `rpl_ddl_high_priority.inc` be put here inline?

Reply:
The testcase also has a rocksdb variant.

Reply:
Oh, I see now.